vitumob/controllers/cart.py

```diff
@@ -1,92 +1,237 @@
+# import json
```

Comment:
You need to clean/revert all these files how they were. For this PR to be mergeable.